### ISSUE TITLE
`Error relocating /usr/lib/libluv.so.1: symbol not found` in Alpine Linux WSL

### ISSUE THREAD

**Issue by cindrmon** [neovim/neovim#24992]
### Problem

I don't know if it is my fault for not knowing much going on with this, but I just installed neovim on Alpine Linux on WSL and I keep on getting these errors when running neovim:
```
-> % nvim
Error relocating /usr/lib/libluv.so.1: uv_get_available_memory: symbol not found
Error relocating /usr/lib/libluv.so.1: uv_thread_setaffinity: symbol not found
Error relocating /usr/lib/libluv.so.1: uv_cpumask_size: symbol not found
Error relocating /usr/lib/libluv.so.1: uv_metrics_info: symbol not found
Error relocating /usr/lib/libluv.so.1: uv_clock_gettime: symbol not found
Error relocating /usr/lib/libluv.so.1: uv_thread_getcpu: symbol not found
Error relocating /usr/lib/libluv.so.1: uv_thread_getaffinity: symbol not found
-> % 
```

Are there any missing dependencies for neovim that I missed? I tried doing `apk fix` for the following packages and I still end up with this error:
```
- neovim
- libluv
- libuv
- gcc
- g++
- musl
```

### Steps to reproduce

1. Install Alpine Linux on WSL.
2. Install `neovim` on Alpine Linux (`sudo apk add neovim`)
3. Run `nvim`.

### Expected behavior

Expected behaviour is that I get to see the neovim welcome screen.

### Neovim version (nvim -v)

0.9.1-r0

> NOTE: running `nvim -v` also results in the error shown above. The version number was taken from the version number that shows up when installing neovim through the `apk` package manager.

### Vim (not Nvim) behaves the same?

no. 9.0

### Operating system/version

Alpine Linux v3.18 on Windows 10 x86_64

### Terminal name/version

Windows Terminal

### $TERM environment variable

xterm-256color

### Installation

system package manager (apk)

**Comment by omurilo** [comment]
hi bro! try to install libluv=1.44.2.1-r0, for me fix the error.

**Comment by cindrmon** [comment]
> hi bro! try to install libluv=1.44.2.1-r0, for me fix the error.

I see. It now introduces me to this problem. How do I force `apk` to install the specific version of `libluv`? I keep on getting this error:

```
-> % sudo apk add libluv=1.44.2.1-r0
ERROR: unable to select packages:
  libluv-1.45.0.0-r0:
    breaks: world[libluv=1.44.2.1-r0]
-> %
```

**Comment by cindrmon** [comment]
> hi bro! try to install libluv=1.44.2.1-r0, for me fix the error.

I see. It did seem to work, but at first, I keep on getting this error when installing the version you specified:
```-> % sudo apk add libluv=1.44.2.1-r0
ERROR: unable to select packages:
  libluv-1.44.2.1-r3:
    breaks: world[libluv=1.45.0.0-r0]
-> %
```

But then I checked the listing of the available versions of `libluv` using `apk list`, and found out that the version number doesn't work for my case:

```
-> % sudo apk list | grep libluv
libluv-1.44.2.1-r3 x86_64 {lua-luv} (Apache-2.0)
libluv-1.45.0.0-r0 x86_64 {lua-luv} (Apache-2.0)
-> %
```

so I installed `libluv-1.44.2.1-r3` instead, and it works.

So, long story short:

1. Tried `sudo apk add libluv=1.44.2.1-r0`, didn't work.
2. Checked for the closest `1.44.x` version with `sudo apk list | grep libluv`.
3. Installed that instead `sudo apk add libluv=1.44.2.1-r3`
4. Reinstalled neovim `sudo apk fix neovim`
5. Neovim works!

Thank you for the help @omurilo!
